zigbee2mqtt/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.18.1-2
+- ⚠️ **Breaking changes**
+  - `log_output` now required option in `advanced`
+- Fix `log_output` option being ignored
+
+Default zigbee2mqtt options for `log_output`
+```
+advanced:
```

Comment:
I don't think an example should be added into the changelog. Better to either link to official docs or don't mention it if it's seamless for most users.

Reply:
I have removed the example and linked to official docs instead. Correct me if I am wrong but users who did not state log_output option previously would not be able to start the add-on or save a new config without the log_output option.

Reply:
If it's not an optional config, you might be correct. Best to test this before releasing.

Reply:
Can you test also how a migration would work? It's best to check if we break the setup for existing users without this option.

Reply:
> If it's not an optional config, you might be correct. Best to test this before releasing.

This is not an optional config as it requires at least an empty array.


> Can you test also how a migration would work? It's best to check if we break the setup for existing users without this option.

Yep, I have tested this and without this option the add-on would not be able to start and existing users would be prompted the message below.

![Screenshot 2021-03-09 at 2 31 51 AM](https://user-images.githubusercontent.com/4586209/110365372-ebc9b980-807f-11eb-85bd-be5316d084b4.png)